inst/conf/default-ic50-fitSettings.json

```diff
@@ -10,7 +10,10 @@
       "type":"threshold",
       "value":20,
       "activeDoses":2,
-      "operator":"<="
+      "operator":">=",
```

Comment:
I don't fully understand this, but is it on purpose that the operator switched direction here?